tests/conftest.py

```diff
@@ -263,8 +263,12 @@ def inner():
                     'metadata': get_metadata_singlecore()
                 },
                 'pw2wannier': {
-                    'code': code_pw2wannier90,
-                    'metadata': get_metadata_singlecore()
+                    # Use a huge batchsize to force it all at once
```

Comment:
Shouldn't there be at least one test where we actually use more than one batch?

Reply:
Yeah that would make sense

Reply:
Fixed in 598e5d1